packages/core/tests/connections.test.ts

```diff
@@ -32,29 +32,29 @@ describe('connections', () => {
       {
         endpoints: ['rxjs:faber'],
       },
-      getIndySdkModules()
+      { askar: askarModule }
     )
     const aliceAgentOptions = getAgentOptions(
       'Alice Agent Connections',
       {
         endpoints: ['rxjs:alice'],
       },
-      getIndySdkModules()
+      { askar: askarModule }
```

Comment:
Really curious how many utility methods we have for creating test agents ;). Might be good in the future to really go through the tests and utilities and see if we can clean some stuff up.

Reply:
Way tooooo many. I think with removing the indy sdk we have cleared a lot of tech debt, and we can probably remove a lot of it. Maybe create one private test pacakge that has utils to set up an agent for tests, and we can remove the 100s of duplicate utils